testing/tests/DevExpress.ui/defaultOptions.tests.js

```diff
@@ -901,78 +903,134 @@ testComponentDefaults(TagBox,
 );
 
 testComponentDefaults(Toast,
-    [{ platform: 'android' }],
+    [{ platform: 'generic', deviceType: 'desktop' }],
     {
-        position: {
-            at: 'bottom left',
-            my: 'bottom left',
-            of: null,
-            offset: '20 -20'
-        },
-        width: 'auto',
         animation: {
             show: {
-                type: 'slide',
-                duration: 200,
-                from: {
-                    position: {
-                        my: 'top',
-                        at: 'bottom',
-                        of: window
-                    }
-                },
+                type: 'fade',
+                duration: 400,
+                from: 0,
+                to: 1
             },
             hide: {
-                type: 'slide',
-                duration: 200,
-                to: {
-                    position: {
-                        my: 'top',
-                        at: 'bottom',
-                        of: window
-                    }
-                },
-            }
+                type: 'fade',
+                duration: 400,
+                from: 1,
+                to: 0
+            },
         }
     }
 );
 
+const tabletAndMobileAnimation = {
+    show: {
+        type: 'fade',
+        duration: 200,
+        from: 0,
+        to: 1
+    },
+    hide: {
+        type: 'fade',
+        duration: 200,
+        from: 1,
+        to: 0
+    }
+};
+
+const mobileOptions = {
```

Comment:
We should not specify such constants in this file. We test default options for all components here - it's not clear that this "mobileOptions" are related only to the toast. Let's use inline objects